src/Uno.UI/Controls/CommandBar/AppBarButtonRenderer.Android.cs

```diff
@@ -98,6 +96,21 @@ protected override void Render()
 						native.SetTitle(null);
 						break;
 
+					case FontIcon: // not supported
+					case PathIcon: // not supported
+					case SymbolIcon: // not supported
```

Comment:
This is reverting https://github.com/unoplatform/uno/pull/12691

Reply:
Doesn't the effectively achieve the same thing? And it also adds back to the warning log for when an unsupported IconElement is used?

Reply:
The thing is that IconElement is now supported. They work through the `case FrameworkElement`, that's why I merged the two switch statements using `iconOrContent`

Reply:
Omg you're right! Ok so it just uses the wrapper, oh wow I can't believe I didn't realize we could do that!

Alright I will keep this and also make a PR in Toolkit for NavigationBar also handles this!